tests/multigrid_evolve_test.cpp

```diff
@@ -280,6 +303,16 @@ void setMultigridParam(QudaMultigridParam &mg_param) {
     mg_param.coarse_solver_tol[i] = coarse_solver_tol[i];
     mg_param.coarse_solver_maxiter[i] = coarse_solver_maxiter[i];
 
+     // Basis to use for CA-CGN(E/R) coarse solver
```

Comment:
clang-format suggested the following change:


```
-     // Basis to use for CA-CGN(E/R) coarse solver
+    // Basis to use for CA-CGN(E/R) coarse solver
```

<!-- identifier: clang-format -->
<!-- File: tests/multigrid_evolve_test.cpp -->